src/coreclr/jit/compiler.cpp

```diff
@@ -4944,6 +4945,16 @@ void Compiler::compCompile(void** methodCodePtr, uint32_t* methodCodeSize, JitFl
         }
     }
 
+    //
+    // Compute reachability and remove unreachable blocks, if any.
+    //
+    auto computeReachability = [this]() {
+
+        EnsureBasicBlockEpoch();
+        fgComputeReachability(/* computeDoms */ false, /* doRenumbering */ false);
+    };
+    DoPhase(this, PHASE_COMPUTE_REACHABILITY, computeReachability);
```

Comment:
I would use a different local name and a different phase name (reusing a phase name likely messes up some stats computations).